sale_order_product_recommendation/wizards/sale_order_recommendation.py

```diff
@@ -385,3 +362,22 @@ def _get_unit_price_from_pricelist(self):
             product_currency=self.currency_id,
         )
         return price_unit
+
+    def _prepare_update_so_line(self):
```

Comment:
Why changing the order of these methods increasing the diff or make difficult to review them?

Reply:
And maybe the names of the methods should be updated accordingly the changes:

```suggestion
    def _prepare_update_so_line_vals(self):
```